src/client/datascience/jupyter/debuggerVariables.ts

```diff
@@ -278,7 +293,7 @@ export class DebuggerVariables extends DebugLocationTracker
         }
     }
 
-    private async getFullVariable(variable: IJupyterVariable): Promise<IJupyterVariable> {
+    public async getFullVariable(variable: IJupyterVariable): Promise<IJupyterVariable> {
```

Comment:
Made this public because `getMatchingVariable` requires only a name, which it uses to lookup a partial `IJupyterVariable` amongst its known variables, and passes that to `getFullVariable`. This doesn't work at all when opening the data viewer from the debug variable context menu, because at that point DebuggerVariables doesn't know about any of our variables yet.